src/utilities.ts

```diff
@@ -177,15 +177,15 @@ export class Utilities {
     }
 
     /**
-     * Generates a folder path matching original path in the temp location and returns the path with the file included
+     * Generates a folder path matching original path in the temp location and returns the path with the file included.
+     * The function will always return the same output for the same given input.
      * @param filePath original file path, to recreate in the temp location
-     * @param hashValue optional hash value to add to the path
      */
-    public static generateTempPath(filePath: string, hashValue?: string): string {
+    public static generateTempPath(filePath: string): string {
```

Comment:
Why is this not a URI and return a URI

Reply:
Because there is a lot of other code that still uses strings. The new extension that Jeff is working on will be ALL Uri :)

---
In reply to: [418139508](https://github.com/microsoft/sarif-vscode-extension/pull/276#discussion_r418139508) [](ancestors = 418139508)